### PR TITLE
login and navigate to Tunnel Main before each e2e test

### PR DESCRIPTION
**Pull request by Hugo-Hontans**
- [x] Create a function to login and create a new movie in the beforeEach of each tests
- [x] Run all the movie-tunnel e2e tests
- [x] Add movie-tunnel spec 11 - file and links
- [x] Add global timeout in cypress.json

### CODE REVIEW

File: apps/catalog/catalog-e2e/src/integration/dashboard/movie-tunnel-2.spec.ts [hunk 1]
```diff
@@ -27,123 +21,114 @@ const AWARDS = ['Opening Film', 'Nominee Best New Documentary Film'];
 const PREMIERE = 'World';
 const YEAR = '2007';
 
-const MOVIE_ID = 'P5ErzmOtap9ju9X8rWvd'; // Empty movie
-
 beforeEach(() => {
   cy.clearCookies();
```
Comment (lsenta) [inline apps/catalog/catalog-e2e/src/integration/dashboard/movie-tunnel-2.spec.ts]: Nice, could you add this part to the utils?
```
  cy.clearCookies();
  cy.clearLocalStorage();
  cy.visit('/auth');
  cy.viewport('ipad-2', 'landscape');
```

could be something like `clearDataAndPrepareTest()`

File: apps/catalog/catalog-e2e/src/support/pages/dashboard/TitlesListPage.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import StartTunnelPage from "./StartTunnelPage";
 
 export default class TitlesListPage {
   constructor() {
-    cy.get('catalog-title-list', { timeout: 5000 });
+    cy.get('catalog-title-list', { timeout: 10000 });
```
Comment (lsenta) [inline apps/catalog/catalog-e2e/src/support/pages/dashboard/TitlesListPage.ts]: what about having a large global / default timeout? https://docs.cypress.io/guides/references/configuration.html#Timeouts

File: apps/catalog/catalog-e2e/src/integration/dashboard/movie-tunnel-2.spec.ts (hunk 1 above)
Comment (Hugo-Hontans) [reply apps/catalog/catalog-e2e/src/integration/dashboard/movie-tunnel-2.spec.ts]: sure

File: apps/catalog/catalog-e2e/src/support/pages/dashboard/TitlesListPage.ts (hunk 2 above)
Comment (Hugo-Hontans) [reply apps/catalog/catalog-e2e/src/support/pages/dashboard/TitlesListPage.ts]: Yes sure, good idea

File: apps/catalog/catalog-e2e/src/support/pages/dashboard/TunnelMainPage.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import TunnelStorylinePage from "./TunnelStorylinePage";
 
 export default class TunnelMainPage {
   constructor() {
-    cy.get('catalog-movie-tunnel-main', { timeout: 5000 });
+    cy.get('catalog-movie-tunnel-main', { timeout: 10000 });
```
Comment (lsenta) [inline apps/catalog/catalog-e2e/src/support/pages/dashboard/TunnelMainPage.ts]: not needed anymore

**Pull request merged**